src/WinRT.Runtime/TypeExtensions.cs

```diff
@@ -155,23 +149,41 @@ public static Type GetGuidType(this Type type)
             }
 #endif
 
-            Type vftblType = helperType.GetNestedType("Vftbl");
-            if (vftblType is null)
-            {
-                return null;
-            }
-            if (helperType.IsGenericType && vftblType is object)
+#if NET8_0_OR_GREATER
+            [RequiresDynamicCode(AttributeMessages.MarshallingOrGenericInstantiationsRequiresDynamicCode)]
+#endif
+#if NET
+            [DynamicDependency(DynamicallyAccessedMemberTypes.All, "ABI.Windows.Foundation.IAsyncActionWithProgress`1+Vftbl", "Microsoft.Windows.SDK.NET")]

```

Comment:
You are missing the generic collections from [here](https://github.com/microsoft/CsWinRT/blob/dc4610e557a85b11d4d5b22dc1957fab36c3e5bc/src/WinRT.Runtime/Projections.cs#L76-L84)

Reply:
Fixed in https://github.com/microsoft/CsWinRT/pull/1521/commits/9d234477b577cd3089b7d0ef7160cacd718ed4c2! 🙂